tsconfig.json

```diff
@@ -5,10 +5,14 @@
         "noImplicitAny": false,
         "sourceMap": false,
         "declaration": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node", 
+        "typeRoots": [ "node_modules/@types" ]
     },
     "files": [
         "index.ts",
         "sample.ts"
+    ], 
+    "exclude": [
```

Comment:
You might want to diff the before/after to make sure the _build folder still looks basically like you expect after your PR is applied.